main.go

```diff
@@ -143,7 +143,7 @@ func main() {
 	exitQuick := flag.Bool("q", false, "exit immediately")
 	showVersion := flag.Bool("v", false, "show version")
 	when := flag.Int64("when", 0, "time in seconds since unix epoch (disables -w)")
-	doSearch := flag.Bool("list", false, "list zones by name")
+	doSearch := flag.Bool("list [filter]", false, "list zones by name")
```

Comment:
Oops, meant to put `[filter]` on the comment not the name, will fix after the preceding PRs are merged.

```go
doSearch := flag.Bool("list", false, "[filter] list or search zones by name")
```

Reply:
Fine, NBD. It's a trivial fix. 👍🏻